packages/discovery-styles/scss/components/search-results/_search-results.scss

```diff
@@ -171,15 +171,28 @@ $result-table-fade-offset: calc(#{$result-element-max-height} - #{$result-table-
   color: $gray-60;
   font-size: $spacing-sm;
   display: flex;
-  align-items: center;
```

Comment:
this is inside of `.#{$prefix}--search-result__footer` without aligning our collection and title name, it allows either one to grow to multiple lines in size, without the other one jumping down to stay aligned vertically